rds/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  profile = var.profile
```

Comment:
Do we need to set profile. As we discussed its fine to exclude from here.

Reply:
It is optional

Reply:
https://github.com/variant-inc/lazy-terraform/pull/19/files#diff-44dfd6f3682d3556fac34cf42e79304547e565b8a7861937b099d146c8d236a0R1-R4